### PR TITLE
Add knapsack exercise

### PR DESCRIPTION
**Pull request by keiravillekode**

### CODE REVIEW

**Comment by keiravillekode** [comment]
Naming and arguments are taken from the [Scheme](https://github.com/exercism/scheme/blob/main/exercises/practice/knapsack/knapsack.scm) track.

File: config.json [hunk 1]
```diff
@@ -659,6 +659,14 @@
...
         "prerequisites": [],
         "difficulty": 5
+      },
+      {
+        "slug": "knapsack",
+        "name": "Knapsack",
+        "uuid": "41029de7-58a6-431b-9f82-6d4603604fb3",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```
Comment (BNAndras) [inline config.json:669]: I think the difficulty should be bumped. Racket marks this an easy exercise (3), Common Lisp as medium (7) and Scheme as hard (8).

File: exercises/practice/knapsack/.meta/example.el [hunk 2]
```diff
@@ -0,0 +1,23 @@
+;;; knapsack.el --- Knapsack (exercism)  -*- lexical-binding: t; -*-
+
+;;; Commentary:
+
+;;; Code:
+
+(require 'cl-lib)
+
+(defun knapsack (capacity weights values)
```
Comment (BNAndras) [inline exercises/practice/knapsack/.meta/example.el:9]: Did the exercise generator use `knapsack` for the function name? Looking at problem specifications, It should be `maximum-value`.

Also, the current argument list clashes with the instructions. The docs indicate the items should be a sequence of items with both weight and value stored together. I'd recommend taking a look at the Common Lisp implementation of this exercise.

I'd prefer the function to also be named maximumValue as indicated in problem-specificatins

**Review by BNAndras — CHANGES_REQUESTED** [review]

File: config.json (hunk 1 above)
Comment (keiravillekode) [reply config.json]: Corrected.

File: exercises/practice/knapsack/.meta/example.el (hunk 2 above)
Comment (keiravillekode) [reply exercises/practice/knapsack/.meta/example.el]: I had changed the name and argument list to follow the Scheme track.

Now updated to follow problem specifications / Common Lisp.

**Review by BNAndras — APPROVED** [review]

**Pull request merged**